lib/entities/scheduled-action.ts

```diff
@@ -57,6 +57,13 @@ export type ScheduledActionSysProps = {
   updatedBy: Link<'User'> | Link<'AppDefinition'>
 }
 
+export const ScheduledActionReferenceFilters =
```

Comment:
could we make this an enum? Just to make easier for users. 

You can also export it here https://github.com/contentful/contentful-management.js/blob/aab92cea7a2829cf2887879316a281f5b5e98c50/lib/export-types.ts#L0-L1

Reply:
We might introduce new filters in the future as well